lib/truffle/bigdecimal.rb

```diff
@@ -65,7 +65,7 @@ def self.limit(limit = nil)
     Thread.current[:'BigDecimal.precision_limit'] ||= 0
     if limit
       raise ArgumentError, 'requires key to be Fixnum' unless limit.is_a? Fixnum
-      old                                           = Thread.current[:'BigDecimal.precision_limit']
+      old = Thread.current[:'BigDecimal.precision_limit']
```

Comment:
A great example of imho too much spacing.

Reply:
What was it aligning with? It looks like someone just left their hand on the keyboard.

Reply:
The assignment below:
```ruby
       Thread.current[:'BigDecimal.precision_limit'] = limit
```